ReactWindows/ReactNative/Modules/WebSocket/WebSocketModule.cs

```diff
@@ -152,6 +152,21 @@ private async void InitializeInBackground(int id, string url, MessageWebSocket w
                     { "id", id },
                 });
             }
+            catch (System.Runtime.InteropServices.COMException ex)
+            {
+#if DEBUG
+                if (url.StartsWith("ws://localhost"))
```

Comment:
It lose the first the exception too. but from the bug, it's better to only avoid it happy every 5 seconds.
I suggest you keep the first exception if possible

Reply:
It doesn't seem necessary to introduce additional state here.